spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/BaseRabbitListenerContainerFactory.java

```diff
@@ -218,4 +220,13 @@ protected ApplicationContext getApplicationContext() {
 		return this.applicationContext;
 	}
 
+	@Override
+	public void setBeanName(String name) {
+		this.beanName = name;
+	}
+
+	public String getBeanName() {
```

Comment:
Please, move these two new method into that `RabbitListenerContainerFactory` interface with `default` impl to avoid a breaking change.

The `getBean()` name also would need some simple Javadoc with a `@since 3.2`.

Reply:
I typed a comment in `RabbitListenerContainerFactory`. Please review and tell me if I miss something